README.md

```
@@ -67,8 +108,32 @@ pip install -r requirements.txt
+
+    ```bash
+    python train.py
+    ```
+    Then, get a forecast directly in your terminal:
+
+    ```bash
+    python cli.py GOOGL
+    ```
+
+## Documentations
+- [Project Methodology Document]()
```

markdownlint-cli2 (0.17.2)

No empty links

(MD042, no-empty-links)